cargo-miri/bin.rs

```diff
@@ -882,7 +891,20 @@ fn phase_rustc(mut args: env::Args, phase: RustcPhase) {
 
     // Run it.
     if verbose {
-        eprintln!("[cargo-miri rustc] {:?}", cmd);
+        eprint!("[cargo-miri rustc] ");
+        let mut envs = HashMap::new();
+        for (key, value) in std::env::vars() {
+            envs.insert(key, value);
+        }
+        for (key, value) in cmd.get_envs() {
+            if let Some(value) = value {
+                envs.insert(key.to_str().unwrap().into(), value.to_str().unwrap().to_owned());
+            }
+        }
+        for (key, value) in envs {
+            eprint!("{key}={value:?} ");
```

Comment:
I think this would be more readable with one var per line.
```suggestion
            eprintln!("{key}={value:?} \\");
```

Reply:
but then I can't copy paste this, which was the whole point XD